src/charm.py

```diff
@@ -91,7 +91,8 @@ def __init__(self, *args: Any) -> None:
         self._prometheus_scrape_relation_name = "metrics-endpoint"
         self._loki_push_api_relation_name = "logging"
         self._hydra_service_command = "hydra serve all"
-        self._log_path = "/var/log/hydra.log"
+        self._log_dir = "/var/log"
+        self._log_path = f"{self._log_dir}/hydra.log"
```

Comment:
Since we are doing it this way, let's use pathlib.Path:
```suggestion
        self._log_dir = Path("/var/log")
        self._log_path = self._log_dir / "hydra.log"
```